timesketch/lib/analyzers/interface.py

```diff
@@ -750,6 +750,28 @@ def __init__(self, index_name):
         if not hasattr(self, 'sketch'):
             self.sketch = None
 
+    def get_fields_list(
+            self, indices=None):
+        """Indices get_mapping ElasticSearch.
```

Comment:
this docstring is just confusing.. .what does `indices get_mapping elastic search` mean?

```
Returns a list of field names in an index.
```

Seems like a more accurate description.